src/locales/zh-Hant.json

```diff
@@ -567,7 +571,7 @@
   "tx_modal_status_complete_text_collect_portfolio": "你的書架",
   "tx_modal_status_complete_text_collect_without_wallet": "你將收到系統的電郵，請跟隨電郵中的指示領取你的 Writing NFT。",
   "tx_modal_status_complete_text_transfer": "已成功發送你的珍藏",
-  "tx_modal_status_complete_title": "付款成功",
+  "tx_modal_status_complete_title": "交易成功！",

```

Comment:
I think we changed this string to avoid the word transaction for the whole collect ux

Reply:
This is the fallback message, collect modal should be using `collect_modal_complete_title`